fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/event/election/fragments/CastVoteFragment.java

```diff
@@ -165,15 +193,10 @@ private void castVote(View voteButton) {
           electionViewModel
               .sendVote(electionId, plainVotes)
               .subscribe(
-                  () -> {
-                    LaoActivity.setCurrentFragment(
-                        getParentFragmentManager(),
-                        R.id.fragment_event_list,
-                        EventListFragment::newInstance);
-                    // Toast ? + send back to election screen or details screen ?
-                    Toast.makeText(requireContext(), "vote successfully sent !", Toast.LENGTH_LONG)
-                        .show();
-                  },
+                  () ->
+                      Toast.makeText(
+                              requireContext(), "vote successfully sent !", Toast.LENGTH_LONG)
+                          .show(),
```

Comment:
Have you checked with fe1 for behaviour coherence ?

Reply:
Yes, we're keeping updated the doc with the inconsistencies so we decided this common behaviour